apps/browser/css/navbar.css

```diff
@@ -38,7 +38,14 @@
   border-radius: 3px;
 }
 
-.navbar > button.disabled {
+.urlbar > button {
+ background-image: url(images/shared/reload-stop-go.png);
```

Comment:
pretty good. the image is a medium-gray, so it stands out on either a light or dark background.
![screen shot 2015-01-07 at 3 44 43 pm](https://cloud.githubusercontent.com/assets/10314059/5653927/490c1508-9684-11e4-919e-55d912ddd58c.png)